scripts/awestruct

```diff
@@ -1,32 +1,35 @@
-#!/usr/bin/env bash
+#!/bin/bash
 
-if [[ "$1" = "--dev" ]] ; then
-    ./scripts/ruby bundle exec awestruct $@
+set -euo pipefail
+
+readonly BUILD_DIR="build"
+readonly ERROR_FILE="${BUILD_DIR}/error.txt"
+
+function err() {
+  echo "$1" >&2
+  exit "${2:-1}"
+}
+
+if [[ "${1:-}" = "--dev" ]] ; then
```

Comment:
This allows use of the variable without screwing with `nounset`. There's no `-` either:

```
       ${parameter:-word}
              Use Default Values.  If parameter is unset or null, the expansion of word is substituted.  Otherwise, the value of
              parameter is substituted.